store/kv/kv.go

```diff
@@ -102,6 +103,17 @@ func measureLow(ctx context.Context, name string, f func() error) {
 		metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets)
 		defer metrics.FdbRequests.Tagged(tags).Histogram("histogram", tally.DefaultBuckets).Start().Stop()
 	}
+	if config.DefaultConfig.Tracing.Enabled {
```

Comment:
Would the following work?

```
err := startTracing(ctx, name)
```

Above assumes no defer needs to be passed up but if you think you need the span later after L117 we can bring it back for Finish() or other things.

The config will be available as it's global so no need to pass that. 

```
func startTracing(ctx context.Context, name string) error {
    if !config.DefaultConfig.Tracing.Enabled {
        return nil
    }
    
    parentSpan, exists := tracer.SpenFromContext(ctx)
    if exists {
        ...
    }
    
    return nil
}
```